sdks/python/apache_beam/io/gcp/bigtableio_test.py

```diff
@@ -17,15 +17,16 @@
 
```

Comment:
probably rename file to bigtableio_it_test to clearly denote that these are end-to-end ITs.

Reply:
Sure, I'll rename and move the new unit tests to a new file